rest/model/listhost.go

```diff
@@ -0,0 +1,31 @@
+package model
+
+import (
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/pkg/errors"
+)
+
+type HostListResults struct {
+	Hosts   []CreateHost
+	Details []APIHostCreateDetail
+}
+
+type APIHostCreateDetail struct {
+	HostId *string `bson:"host_id" json:"host_id"`
+	Error  *string `bson:"error" json:"error"`
+}
+
+func (a *APIHostCreateDetail) BuildFromService(t interface{}) error {
```

Comment:
Did you intentionally not generate this?

Reply:
Honestly I forgot that was an option 😨